content/html/concepts/attributes/attributes.md

```diff
@@ -13,22 +15,46 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Attributes can be added to the opening tag of an HTML element to change its default behavior or provide additional data about it.
 
-Attributes consist of a name and a value in the format `name="value"`:
 
-```pseudo
-<tag name="value"></tag>
+
+# HTML Attributes 
```

Comment:
```suggestion
```
We do not use opening h1 headers in Docs, the Title in the metadata provides that.